core/domain/feedback_services.py

```diff
@@ -179,7 +182,7 @@ def get_last_updated_time(exploration_id):
 
     If this exploration has no threads, returns None.
     """
-    threadlist = get_all_threads(exploration_id, False)
+    threadlist = get_displayable_threads(exploration_id, False)
     return max(
```

Comment:
Yes, that's right. There are two types of feedback threads, those with a suggestion attached, and those without. has_suggestion is a boolean indicating which of these is the case. The maximum here should be taken over all the threads.